pkg/queue/queue.go

```diff
@@ -177,6 +178,15 @@ func (q *Queues) Add(namespace string, name string, uri string,
 		idleWorkers = spec.idleWorkers
 	}
 
+	approxSecondsToProcessOneJob := float64(0)
+	totalEnqueueRate := float64(0)
+	for _, job := range jobs {
```

Comment:
Calculate seconds to process a job based on provided enqueue rates per job. Job enqueue rate must be greater than 0.1.